client/src/components/TaskStatusChart/TaskStatusChart.js

```diff
@@ -30,35 +30,38 @@ const chartLabel = ({
   );
 };
 
-const COLORS = ["#629632", "#F6BE00", "#FF0000"];
+const COLORS = ["#4F7009", "#F6BE00", "#cc0000"];
 
 export const TaskStatusChart = ({ tasksData, chartHeader }) => {
```

Comment:
Minor comment: Is there a way to remove where it says 3 of 3?

![image](https://user-images.githubusercontent.com/24628243/165236175-d1d9297d-6807-42c9-bbf2-13e0d38f6316.png)

Reply:
Resolved: I used CSS to override this style.